### DISCUSSION TITLE
quick Capture on desktop script from documentation

### DISCUSSION THREAD

**Discussion by Gr-Nobel** [chhoumann/quickadd#340] [Q&A]
![image](https://user-images.githubusercontent.com/95166364/204439279-ec3d66a3-ccf0-4daa-9ab1-e1d87054fcd4.png)
I copied this script from documentation. Installed the advanced uri plugin. When I press the hotkeys, it just opens Obsidian but the task command doesn't open.

I pasted the URI link on the explorer title bar it's working.
![image](https://user-images.githubusercontent.com/95166364/204439803-439e09bc-f831-4688-9d0a-921f495d1bf0.png)
![image](https://user-images.githubusercontent.com/95166364/204439811-bdc1f02c-9adf-4fb7-a0ed-a3a17b1e8c20.png)

**Comment by chhoumann** [comment]
I resolved this issue once, but exactly how I did it unfortunately seems to escape me.

However, I have taken to use the [Global Hotkeys](https://github.com/mjessome/obsidian-global-hotkeys) plugin for this.
Updated docs accordingly: https://quickadd.obsidian.guide/docs/Misc/AHK_OpenQuickAddFromDesktop
